viadot/sources/genesys.py

```diff
@@ -322,6 +322,7 @@ def download_all_reporting_exports(
                 "queue_performance_detail_view",
                 "queue_interaction_detail_view",
                 "agent_status_detail_view",
+                "agent_interaction_detail_view",
```

Comment:
It is not connected but please correct typos in line 247: `Al reports were successfully dowonload` -> `Al` and `dowonload`

Reply:
updated!